backend/apps/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+
+from . import views
+
+app_name = 'apps'
```

Comment:
`app_name` doesn't appear to have been used?

Reply:
Or is this a variable that's set here but read elsewhere? I'm seeing that we have `apps.User` and `apps.url`.